packages/store/gas-report.json

```diff
@@ -753,19 +759,19 @@
     "file": "test/StoreCoreGas.t.sol",
     "test": "testHooks",
     "name": "set record on table with subscriber",
-    "gasUsed": 72296
```

Comment:
when they're tiny fluctuations like this, it's usually because new bytecode = new optimizer path = may not be optimized in the exact same way as before